server/.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules/
 npm-debug.log
+uploads/*
+!uploads/.gitkeep 
```

Comment:
Ok, since multer does not create the folder I had to add code to create the folder:

```javascript
var fs = require('fs');
var dir = config.uploads;

if (!fs.existsSync(dir)){
    fs.mkdirSync(dir);
}
```
but I think this code shouldn't go in 'controller/tree/create.js', but maybe somewhere in middleware?